integration/plugin/testdata/example-plugin/go.mod

```diff
@@ -1,6 +1,8 @@
 module example-plugin
 
-go 1.20
+go 1.21.1
```

Comment:
I do not fully comprehend what of this diff when I have merged main into my branch. Is some sort of hiccup from Github?

Reply:
It might be caused by `go mod tidy`. I think it will update the Go version in the mod file(s) to `1.21.1` in this case because the CLI imports `nodetime` from the "ignite-files" repo which [requires](https://github.com/ignite/ignite-files/blob/cb867d7a6af2fc00667bcff6a4a4d6cb32177236/nodetime/go.mod) that Go version.

It's strange that a mod file inside a testdata folder was updated though.

We could lower the dependency in `nodetime` mod file to `1.21.0` which would be right and would match the one used in the CLI.

Reply:
You mean here right https://github.com/ignite/ignite-files/blob/main/nodetime/go.mod ?

Reply:
PR wired here https://github.com/ignite/ignite-files/pull/16